test/integration/suites/join-token/00-setup

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
this file is required?

Reply:
It is currently. It used to serve as both a sentinel that the directory was a test suite and as the first script to be executed. Now that the suites are under a subdirectory, I could probably relax that requirement.... 99-teardown is required though (it is run no matter what)